lib/laa_crime_forms_common/pricing/nsm/calculators/letters_and_calls.rb

```diff
@@ -4,7 +4,7 @@ module Nsm
       module Calculators
         class LettersAndCalls
           class << self
-            def call(claim, rates:)
+            def call(claim, rates: Rates.call(claim))
```

Comment:
As discussed, this is redundant, as this method should never be called by anything other than LaaCrimeFormsCommon::Pricing::Nsm::Calculators::Totals, which explicitly passes in a `rates` argument.